release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -84,6 +84,13 @@ The {product-title} 4.10 installer uses new default component types for installa
 * AWS EC2 M6i instances for both control plane and compute nodes, where available
 * AWS EBS gp3 storage
 
+[id="ocp-4-10-enhancements-to-API-in-installconfig"]
+==== Enhancements to API in install.config
+
+When a user installs OCP on a Baremetal IPI platform, they have nowhere to configure custom network interfaces, such as static IPs or a vLAN to communicate with the Ironic server.
+
+With this feature, for Day 1 install on bare metal only, using a new API change in install-config, the user can customise the network configuration (network-config) per host during the install and provisioning process, including setting static IPs per host and other advanced network configurations.
```

Comment:
Im not sure - but "...network configuration (network-config)..." seemed a little excessive.
are you referring to the new 'networkConfig' field in install-config.yaml? if so you should correct it to the exact capitalization.
if not , are you referring to a different technical element?

Reply:
Updated as suggested.